gulpfile.js

```diff
@@ -0,0 +1,35 @@
+var gulp = require('gulp'),
+  rename = require('gulp-rename'),
+  react = require('gulp-react'),
+  path = require('path'),
+  browserSync = require('browser-sync'),
+  reload = browserSync.reload;
+  
+var paths = {
+  startUrl: '/lesson_1/lesson.html',
+  scripts: ['lesson_*/src/*.js']
+};
+ 
+gulp.task('server', function() {
+  browserSync({
```

Comment:
Both `gulp-connect` and `browser-sync` can run web servers, but `browser-sync` has a little more functionality for developer workflow. Both would work here! Just curious if there was a reason to use one over the other :smile: